util/strings.go

```diff
@@ -94,3 +94,18 @@ func CoalesceString(in ...string) string {
 func CoalesceStrings(inArray []string, inStrs ...string) string {
 	return CoalesceString(CoalesceString(inArray...), CoalesceString(inStrs...))
 }
+
+// StringContainsSliceRegex determines if a string matches a regex in a slice
+func StringContainsSliceRegex(slice []string, item string) bool {
+	if len(slice) == 0 {
+		return false
+	}
+
+	for idx := range slice {
```

Comment:
you can use the notation `idx, sliceItem := range slice {` and then you can use `sliceItem` instead of `slice[idx]` (the only time we don't want to do this is if you're making modifications to slice)